### ISSUE TITLE
Global typographic styles suck

### ISSUE THREAD

**Issue by anthonyshort** [anthonyshort/anthonyshort.me#4]
Comments for the post "Global typographic styles suck"

**Comment by kaelig** [comment]
I have used this approach on a few CMS that required `.wysiwyg` classes for copy blocks, but in the end it generated more clutter and specificity issues. But I see your point.

In _most_ cases, think typography should be at the center of the design, thus becoming the global styles.

Both approaches aren't "bad" or "good", they just are two means to different ends:
- building the UI around the content: global typographic styles
- putting content in a UI: scoped typographic styles

**Comment by anthonyshort** [comment]
I've found that pulling it all out how made everything less messy and specific. But of course, it depends on the project. I don't really believe that the copy block styles are the centre of the design anymore. They are theorically but not in terms of CSS structure. In app layouts with lots of complex UI elements it's much more obvious.

**Comment by kaelig** [comment]
Exactly, good developers should consider both approaches depending on the project. For a blog, global styles may be the best answer, whereas for a web app, scoped styles are more likely to be a better fit.

**Comment by gandhiShepard** [comment]
This makes so much sense.

**Comment by kevinSuttle** [comment]
I feel like just defining copy styles in Sass %placeHolders and extending them in your selectors, vs classes in your HTML, is the most scalable and flexible approach. http://ianstormtaylor.com/oocss-plus-sass-is-t…
